packages/alias/alias.mjs

```diff
@@ -8,17 +8,13 @@ import _get from 'lodash.get';
 const projectRoot = pathToFileURL(`${process.cwd()}/`);
 
 const aliasFieldPaths = {
-  'package.json': 'aliases',
   'tsconfig.json': 'compilerOptions.paths',
 };
```

Comment:
This dict is obsolete: references to `aliasFieldPaths` should be replaced with just `'compilerOptions.paths'`